x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/InstanceTypeService.java

```diff
@@ -20,40 +22,46 @@
 import java.util.zip.GZIPInputStream;
 
 public final class InstanceTypeService {
-
     private InstanceTypeService() {}
 
     private static final class Holder {
         private static final Map<InstanceType, CostEntry> costsPerDatacenter;
```

Comment:
I missed this the last time but the convention is to use to upper-case and use underscores as separators. So this should become:

```suggestion
        private static final Map<InstanceType, CostEntry> COSTS_PER_DATACENTER;
```